de/herunterladen.html

```diff
@@ -33,20 +33,20 @@ <h2>Windows</h2>
 
 <h2>macOS</h2>
 <p>
-Subsurface unterstützt Intel Macs mit macOS 10.12 oder neuer. Aus technischen Gründen bieten wir drei verschiedene DMGs an.
+Subsurface unterstützt Intel Macs mit macOS 10.12 oder neuer. Aus technischen Gründen bieten wir drei verschiedene DMGs an:
 <p>
 <center>
 <a class="face-button" href="https://subsurface-divelog.org/downloads/Subsurface-{{ site.latest }}-10.15+11.0.dmg">
-  <div class="face-primary"><i class="fas fa-file-download"></i></span> macOS 10.15 / 11.0 </div>
+  <div class="face-primary"><i class="fas fa-file-download"></i></span> macOS 10.15 (Catalina) / 11.0 (Big Sur)</div>
   <div class="face-secondary"><i class="fas fa-file-download"></i> .DMG herunterladen</div>
 </a>
 <br/>
 <a class="face-button" href="https://subsurface-divelog.org/downloads/Subsurface-{{ site.latest }}-10.13+14.dmg">
-  <div class="face-primary"><i class="fas fa-file-download"></i></span> macOS 10.13 / 10.14 </div>
+  <div class="face-primary"><i class="fas fa-file-download"></i></span> macOS 10.13 (High Sierra)/ 10.14 (Mojave)</div>
```

Comment:
did you test that locally? That will make the button HUGE - same above for Catalina / Big Sur

Reply:
I guess it's tolerable - but it does feel like it looks a bit odd. Especially on a narrow phone screen.
<img width="784" alt="Screen Shot 2020-12-02 at 9 08 36 AM" src="https://user-images.githubusercontent.com/237471/100906392-fcb1d000-347d-11eb-98ff-6a07ef66d808.png">
simulated phone
<img width="415" alt="Screen Shot 2020-12-02 at 9 10 53 AM" src="https://user-images.githubusercontent.com/237471/100906694-4d292d80-347e-11eb-94fb-02127ff7b953.png">
notice how half the button text is missing

Reply:
maybe the real issue is the "pretty button" that I decided to go with... because of the way it's designed it doesn't allow for multiple lines of text. I can reconsider that choice.

Reply:
No, I just looked at the source code in emacs. Maybe it's just me, but I am very bad with the version numbers and find the names much more memorable. That's why I added them. But feel free to ignore that hunk if it ruins the layout. 

Just pushed another version with one more space (after High Sierra).

Reply:
Funny thing... the names mean nothing to me at all. I mean, I have no idea what the name is of the version that I'm running. It's 10.15, that I know.
But I get your point.
I've spent the last hour trying to come up with a visually pleasing way to do this (because it is, objectively, better to have the names as well)